### ISSUE TITLE
Twitter Dataset

### ISSUE THREAD

**Issue by igorbrigadir** [neulab/covid19-datashare#5]
This isn't totally public because you have to have an approved app, but Twitter just released an endpoint that provides the COVID-19 firehose: all tweets matching this list  https://developer.twitter.com/en/docs/labs/covid19-stream/filtering-rules can be streamed live: https://blog.twitter.com/developer/en_us/topics/tools/2020/covid19_public_conversation_data.html

Since the data is live streamed, the dataset is not complete yet - but i imagine it might be useful to collect the tweet IDs as a dataset for this project too! Social media posts would also make a good addition.

**Comment by wildlewis** [comment]
I like the idea of social content being added. Twitter has some restrictions on use, so that may affect the utility and shareability of the data. Although maybe Twitter can be convinced to change their data sharing restrictions?

**Comment by igorbrigadir** [comment]
Yeah the restrictions are documented here: https://developer.twitter.com/en/developer-terms/agreement-and-policy and there are additional restrictions on use cases here: https://developer.twitter.com/en/developer-terms/more-on-restricted-use-cases

basically you're allowed to share Tweet IDs only, and must re-download when using them again - this ensures that anyone in the dataset retains some control over how their tweets will be used - deleted tweets will simply fail to download again. From a data sharing perspective, this means distributing additional code or instructions for "hydrating" the data like https://github.com/DocNow/twarc#hydrate for example.

It's not certain if the output of this special COVID-19 Stream will be any different - maybe there are additional freedoms or restrictions.
